atomic_reactor/plugins/post_export_operator_manifests.py

```diff
@@ -118,8 +118,15 @@ def run(self):
                            IMG_MANIFESTS_PATH, ex)
             self.log.error(msg)
             raise RuntimeError('%s %s' % (msg, ex))
+
+        except Exception as ex:
+            raise RuntimeError('%s' % ex)
+
         finally:
-            self.tasker.d.remove_container(container_id)
+            try:
+                self.tasker.d.remove_container(container_id)
+            except Exception as ex:
```

Comment:
Do we want to proceed execution here after **any** sort of exceptions (instead of APIError and ReadTimeoutError only)?

Reply:
well it actually isn't that critical that remove_container fails, as long as it was able to get that archive

I can change commit message, but still we are now catching Exception for get_archive as well